### ISSUE TITLE
Request to register iCanHaz with Bower

### ISSUE THREAD

**Issue by ataki** [HenrikJoreteg/ICanHaz.js#57]
I'm currently using Bower as a package manager for front-end dependencies (https://github.com/twitter/bower). 

It's got a sizeable list of templating scripts registered such as Mustache / Handlebars, and it would be nice to be able to use it to manage iCanHaz.js instead of upgrading manually.

Registration process is fairly simple imo (http://twitter.github.com/bower/ toward the bottom) and anyone can do it, but I wanted to check here first. If it's ok I'd like to register it.

Cheers

**Comment by HenrikJoreteg** [comment]
registered as icanhaz

**Comment by birgitta410** [comment]
We are having trouble installing the latest ICanHaz version with bower. Bower always installs v0.6.1, and we cannot figure out for the life of us why that is.

We tried:
`bower install icanhaz
bower install git@github.com:HenrikJoreteg/ICanHaz.js.git
bower install https://github.com/HenrikJoreteg/ICanHaz.js.git`
They all resulted in v0.6.1

We also tried
`bower install https://github.com/HenrikJoreteg/ICanHaz.js.git#10.0.2`
But that results in an error "- ICanHaz.js Could not find tag satisfying: ICanHaz.js#10.0.2"

**Comment by birgitta410** [comment]
P.S.: We forked the repository and tagged it with "v0.10.2", that helped... Seems like Bower relies on tags in git to resolve the version. Still unclear why it always tried 0.6.1 though, because that's not even your latest tag...
Our forked, tagged repo that worked:
https://github.com/thekua/ICanHaz.js

**Comment by ducin** [comment]
Still it's strange. Bower shows there are 2 components available:

```
tomasz@tomasz-lenovo-ideapad-Y530:~/Development$ bower search icanhaz
Search results:

    icanhaz git://github.com/henrikjoreteg/icanhaz.js
    icanhazjs git://github.com/HenrikJoreteg/ICanHaz.js.git
```

Anyway, they refer to the same github repository. When I want to use the first one, I get following error:

```
tomasz@tomasz-lenovo-ideapad-Y530:~/Development$ bower install icanhaz --save
bower icanhaz#*             not-cached git://github.com/henrikjoreteg/icanhaz.js.git#*
bower icanhaz#*                resolve git://github.com/henrikjoreteg/icanhaz.js.git#*
bower icanhaz#*                ECMDERR Failed to execute "git ls-remote --tags --heads git://github.com/henrikjoreteg/icanhaz.js.git", exit code of #128

Additional error details:                                                                                                                                        
fatal: remote error:                                                                                                                                             
  Repository not found.
```

But when I try the second one, it's ok. Why is that?

**Comment by cvn** [comment]
I'm having the same issues as @tkoomzaaskz . It appears that the `icanhaz` package uses all lowercase characters in it's urls, while  `icanhazjs` uses mixed case urls, which seem to match how the repo is actually named. Anyone know how to unregister a package? Having the broken `icanhaz` in the registry is confusing.

**Comment by HenrikJoreteg** [comment]
I don't use bower, so I'm not sure, sorry.

**Comment by ducin** [comment]
Yep, I agree, it is confusing a lot. It'd be good for icanhaz itself to
make its bower package clean.

You may unregister a package by sending email directly to bower guys, as
stated in the bower homepage: "There is no direct way to unregister a
package yet. For now, you can request a package be
unregisteredhttps://github.com/bower/bower/issues/120".
I think we should decide here how the icanhaz bower package should be like
and send the email.

**Comment by HenrikJoreteg** [comment]
I opened a request: https://github.com/bower/bower/issues/120
